src/jsx/repo.jsx

```diff
@@ -292,3 +301,26 @@ var Repo = React.createClass({
     }
   }
 });
+
```

Comment:
I'd rather us move to lodash rather than polyfill this stuff. It just makes it more work for us. I don't care about the extra minimal load time for lodash. Go ahead and add it as a bower dep and include it in the main HTML file. Then you can simplify the above logic to:

```
var hasEnhancementLabel = _.some(item.labels, function(label) {
  return (label.name === 'enhancement');
});
```

which is just so much nicer :)